library/src/main/java/com/bumptech/glide/load/model/ModelEquity.java

```diff
@@ -0,0 +1,16 @@
+package com.bumptech.glide.load.model;
+
+/**
+ * An optional interface that models can implement to handle difference in equity
+ * between the model and the target.
+ */
+public interface ModelEquity {
```

Comment:
I might just call this Model. We reference Model all over the place in the documentation and it's plausible we might expand this interface or make it required in a future version. What do you think?

Reply:
Yes Model is Ok if there's plan to expand.